composer.json

```diff
@@ -13,40 +13,66 @@
   },
   "minimum-stability": "stable",
   "require": {
-    "php": "~5.6|~7.0",
-    "symfony/symfony": "3.4.*",
-    "twig/twig": "1.41.0",
-    "twig/extensions": "^1.5",
-    "symfony/assetic-bundle": "~2.3",
-    "symfony/monolog-bundle": "^3.1.0",
-    "sensio/distribution-bundle": "^5.0",
-    "sensio/framework-extra-bundle": "~3.0",
-    "incenteev/composer-parameter-handler": "~2.0",
-    "nelmio/security-bundle": "~1.4",
-    "mopa/bootstrap-bundle": "^3.2",
-    "twbs/bootstrap": "^3.3.0",
+    "php": "~7.2",
     "fortawesome/font-awesome": "~4.2.0",
-    "jms/translation-bundle": "~1.3.0",
     "guzzlehttp/guzzle": "^6",
+    "incenteev/composer-parameter-handler": "~2.0",
+    "jms/translation-bundle": "~1.3.0",
+    "mopa/bootstrap-bundle": "^3.2",
+    "mopa/composer-bridge": "~1.5",
+    "mpdf/mpdf": "^7.0",
+    "nelmio/security-bundle": "~1.4",
+    "openconext/monitor-bundle": "^1.0",
+    "sensio/distribution-bundle": "^5.0",
+    "sensio/framework-extra-bundle": "~3.0",
     "surfnet/stepup-bundle": "~4.0",
     "surfnet/stepup-middleware-client-bundle": "^3.0",
     "surfnet/stepup-saml-bundle": "^4.1",
-    "surfnet/stepup-u2f-bundle": "dev-develop",
-    "mopa/composer-bridge": "~1.5",
-    "openconext/monitor-bundle": "^1.0",
-    "mpdf/mpdf": "^7.0"
+    "surfnet/stepup-u2f-bundle": "3.4.0",
+    "symfony/assetic-bundle": "~2.3",
+    "symfony/monolog-bundle": "^3.1.0",
+    "symfony/symfony": "3.4.*",
+    "twbs/bootstrap": "^3.3.0",
+    "twig/extensions": "^1.5",
+    "twig/twig": "1.41.0"
   },
   "require-dev": {
+    "jakub-onderka/php-parallel-lint": "^1.0",
     "matthiasnoback/symfony-config-test": "^2.2",
     "mockery/mockery": "~1.0",
     "phpmd/phpmd": "^2.6",
     "phpunit/phpunit": "^5.7",
+    "roave/security-advisories": "dev-master",
     "sebastian/exporter": "~2.0",
     "sebastian/phpcpd": "^2.0",
     "squizlabs/php_codesniffer": "^3.4",
     "symfony/phpunit-bridge": "^3.0"
   },
   "scripts": {
+    "test": [
+      "@lint",
+      "@phpmd",
+      "@phpcs",
+      "@phpcpd",
+      "@phpunit",
+      "@behat",
+      "@security-tests"
```

Comment:
While at it, why not add the docheader qa check? See stepup-tiqr for an implementation example.

Reply:
Sure, but lets do it in a different PR. I do not want to include too much unrelated stuff and only stick to the php7.2 upgrade